src/main/osd/osd.h

```diff
@@ -299,6 +299,7 @@ typedef struct osdConfig_s {
     uint8_t logo_on_arming_duration;          // display duration in 0.1s units
     uint8_t camera_frame_width;               // The width of the box for the camera frame element
     uint8_t camera_frame_height;              // The height of the box for the camera frame element
+    bool stat_show_cell_value;
```

Comment:
The new element needs to be added to the end of the structure otherwise you corrupt the parameter group.

Reply:
Thanks. It was last..., but after merging this file, the element lost it's index. Sorry about that.
Done.